cpp/lsst/kht/khtContinued.py

```diff
@@ -0,0 +1,26 @@
+
+import numpy as np
+from .pykht import find_satellites as fs
```

Comment:
you only call this once, so I would just `from .pykht import find_satellites` instead of abbreviating it to `fs`.